extension/src/popup/views/Account/styles.scss

```diff
@@ -44,7 +44,7 @@
         stroke: var(--sds-clr-base-01);
         width: 1rem;
         height: 1rem;
-        stroke: var(--sds-clr-gray-11);
+        stroke: var(--sds-clr-gray-12);
```

Comment:
fixing the button color of the `Send` and `...` buttons on Account

Reply:
<img width="384" alt="Screenshot 2024-11-05 at 10 45 41 AM" src="https://github.com/user-attachments/assets/1c076d27-67c6-43f7-9a87-9aebdff1f9b5">

Hmm gray-11 loooks like this for me 🤔